lapce-proxy/src/plugin.rs

```diff
@@ -183,6 +187,65 @@ impl PluginCatalog {
         Ok(plugin)
     }
 
+    fn get_plugin_env(
```

Comment:
Given the amount of possible error cases here, it might be better for this function to return a `Result` so the caller can better tell what has happened.

Reply:
I think this function could be a method of `PluginDesc` as I don't see any use of `self`.